src/apps/actions.js

```diff
@@ -0,0 +1,25 @@
+export const NEW_BUNDLE = 'NEW_BUNDLE';
+export const NEW_ERROR = 'NEW_ERROR';
+export const FETCH_BUNDLE_ERROR = 'FETCH_BUNDLE_ERROR';
+
+export function newBundle(bundle, inputs) {
+    return {
+        type: NEW_BUNDLE,
```

Comment:
Its a really good idea, I'll switch to it future work. Not going to do it in this pr.